arlpy/bf.py

```diff
@@ -12,6 +12,7 @@
 
 import numpy as _np
 import scipy.signal as _sig
+import numpy.linalg as _LA
```

Comment:
Stylistic suggestion: _la (CAPS used for constants)

Reply:
Also I just noticed that we are already using `_np.linalg` in places, and so if you don't have many instances of `_LA`, maybe drop the `import` and just use `_np.linalg` directly?

Reply:
(completely optional -- I don't mind either way)